server/server_test.go

```diff
@@ -291,14 +291,14 @@ func TestMain_FrameRestore(t *testing.T) {
 
 	// Create frames.
 	client := m0.Client()
-	if err := client.CreateIndex(context.Background(), "x", pilosa.IndexOptions{}); err != nil && err != pilosa.ErrIndexExists {
+	if err := client.CreateIndex(context.Background(), "i", pilosa.IndexOptions{}); err != nil && err != pilosa.ErrIndexExists {
```

Comment:
yes, it was doing a restore on an empty node for an index that it didn't think it was responsible for. by changing the index name to `i`, the partition lined up with the node as the test expected.